pkg/connections/connections.go

```diff
@@ -19,15 +19,14 @@ func NewConnectionsManager() *ConnectionsManager {
 }
 
 func getKey(connections *eventsv1alpha1.EventConnections) string {
-	return connections.APIVersion + "/" + connections.Kind + "/" + connections.Namespace + "/" + connections.Name
+	return connections.Namespace + "/" + connections.Name
```

Comment:
When grabbing `EventConnections` from the `event.DeleteEvent` object, the `APIVersion` and `Kind` fields are not set causing `RemoveConnections` to fail since `getKey` to return something like `//default/example-connections`. Instead of hardcoding these values, I figured removing the fields was a better choice. Kind should always be `EventConnections` and, I assume, APIVersion will be whatever version of the operator they have installed.

Let me know if you have a better idea of how to handle this.

Reply:
Agree namespace/name should work.